routes/index.js

```diff
@@ -370,13 +370,16 @@ Routes.prototype.proxyRouteOpenhab = function (req, res) {
     delete requestHeaders['x-forwarded-for'];
     delete requestHeaders['x-forwarded-proto'];
     delete requestHeaders['connection'];
-    requestHeaders['host'] = req.headers.host || system.getHost() + ':' + system.getPort();
+    requestHeaders['host'] = req.headers["x-forwarded-host"] || system.getHost() + ':' + system.getPort();
```

Comment:
if you notice on line 384 the code resets the host value, so this change is not doing anything when we are doing a full proxy through the "home" domain.  I understand why you changed this, but did you test it out to ensure it worked?   I agree we should be setting the forwarded host ip,  i don't know exactly why the author did it this way.